source/code/plugins/agent_maintenance_script.rb

```diff
@@ -262,11 +195,31 @@ def apply_dsc_endpoint(server_resp)
       return dsc_endpoint
     end
 
+    # Update the topology and telemetry request frequencies
+    def apply_query_intervals(server_resp)
```

Comment:
need to modify now that ODS topology response will give both topology query interval and telemetry query interval.